main/test/ca/uwaterloo/flix/language/phase/TestLexer.scala

```diff
@@ -0,0 +1,298 @@
+  }
+
+  test("LexerError.UnterminatedBuiltIn.01") {
+    val input =
+      s"""
+         |def f(): Int32 = $$BUILT_IN; 42
+       """.stripMargin
+    val result = compile(input, Options.TestWithLibNix)
+    expectError[LexerError.UnterminatedBuiltIn](result)
+  }
+
+  test("LexerError.UnterminatedBuiltIn.02") {
+    val input =
+      s"""
+         |def f(): Unit = $$BUILT_IN
+       """.stripMargin
+    val result = compile(input, Options.TestWithLibNix)
+    expectError[LexerError.UnterminatedBuiltIn](result)
+  }
+
+  test("LexerError.UnterminatedBuiltIn.03") {
+    val input =
+      s"""
+         |def f(): Unit = $$BUILT_/*IN*/$$
+       """.stripMargin
+    val result = compile(input, Options.TestWithLibNix)
+    expectError[LexerError.UnterminatedBuiltIn](result)
+  }
+
+  test("LexerError.UnterminatedBuiltIn.04") {
+    val input =
+      s"""
+         |def f(): Unit = $$BUILT_IN""".stripMargin
+    val result = compile(input, Options.TestWithLibNix)
+    expectError[LexerError.UnterminatedBuiltIn](result)
+  }
+
+  test("LexerError.UnterminatedChar.01") {
+    val input =
+      s"""
+         |def f(): Char = 'a
+       """.stripMargin
+    val result = compile(input, Options.TestWithLibNix)
+    expectError[LexerError.UnterminatedChar](result)
+  }
+
+  test("LexerError.UnterminatedChar.02") {
+    val input =
+      s"""
+         |def f(): Char = '\uffff
+       """.stripMargin
+    val result = compile(input, Options.TestWithLibNix)
+    expectError[LexerError.UnterminatedChar](result)
+  }
+
+  test("LexerError.UnterminatedChar.03") {
+    val input =
+      s"""
+         |def f(): Char = 'a/* This is a block-comment */'
+       """.stripMargin
+    val result = compile(input, Options.TestWithLibNix)
+    expectError[LexerError.UnterminatedChar](result)
+  }
+
+  test("LexerError.UnterminatedChar.04") {
+    val input =
+      s"""
+         |def f(): Char = '/* This is a block-comment */a'
+       """.stripMargin
+    val result = compile(input, Options.TestWithLibNix)
+    expectError[LexerError.UnterminatedChar](result)
+  }
+
+  test("LexerError.UnterminatedChar.05") {
+    val input =
+      s"""
+         |def f(): Char = 'a""".stripMargin
+    val result = compile(input, Options.TestWithLibNix)
+    expectError[LexerError.UnterminatedChar](result)
+  }
+
+  test("LexerError.UnterminatedInfixFunction.01") {
+    val input =
+      s"""
+         |def f(): Int32 = 1 `add 2
+       """.stripMargin
+    val result = compile(input, Options.TestWithLibNix)
+    expectError[LexerError.UnterminatedInfixFunction](result)
+  }
+
+  test("LexerError.UnterminatedInfixFunction.02") {
+    val input =
+      s"""
+         |def f(): Int32 = 1 `add/*this is a block comment*/` 2
+       """.stripMargin
+    val result = compile(input, Options.TestWithLibNix)
+    expectError[LexerError.UnterminatedInfixFunction](result)
+  }
+
+  test("LexerError.UnterminatedInfixFunction.03") {
+    val input =
```

Comment:
Meh. Why so ugly? It could simply be on one line now. val input = "myStringHere".

Reply:
You mean:
```scala
val input = s"1 `add 2"
```
I was just trying to be consistent with the formatting I found in the other test files. I will convert to onelines where possible.